### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by NinjaLegendmc** [prateek-chaubey/YTPro#67]
![Screenshot_2024_0920_222633](https://github.com/user-attachments/assets/141697c8-409e-4fcf-9c3a-495a576a0955)

This bugs happened sometimes

**Comment by axqr** [comment]
There is another bug, Hide Shorts option isn't working (I am on mobile), closed and reopened done every tricks, but not working. @prateek-chaubey

**Comment by yuipp** [comment]
Same but I just rotate phone and it's ok fullscreen

**Comment by prateek-chaubey** [comment]
Please share more information regarding the issue

**Comment by NinjaLegendmc** [comment]
When I set the video mode to full screen it just bug like top image but if I rotate the phone it just fixed the issue but it is annoying to rotate the screen always

**Comment by prateek-chaubey** [comment]
Can you provide some more information 
1. Android Version 
2. App Version that you are using 
3. The JS Version (you can check it from YTPRO settings, aside the YTPRO header)
4. Does this bug shows up for every video quality settings or only a few

**Comment by NinjaLegendmc** [comment]
1. Android version: 14
2. This bug appear every video when try to do full screen 
3. Yt pro app version latest 3.5 
Unfortunately I want to send video how the bug happens but GitHub sending me errors when uploading the video

**Comment by prateek-chaubey** [comment]
You can send me on my telegram [@th30neand0nly](https://t.me/th30neand0nly)

**Comment by prateek-chaubey** [comment]
Yeah , as mentioned in  [#74](https://github.com/prateek-chaubey/YTPro/issues/74#issuecomment-2628244771)

> maybe because of the injected elements in the player for skipping sponsors, and when they don't get injected property it shows this side effect,
> or maybe some other reason lol
> i shall be fixing this in the next update 

As i have fixed the element issue , it should work properly now
